File: .API/Cloud/Member.cs
// Decompiled with JetBrains decompiler
// Type: CloudX.Shared.Member
// Assembly: CloudX.Shared, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null
// MVID: 2635624C-5F24-4EFB-ACD1-7E9C1349B2F5
// Assembly location: F:\SteamLibrary\steamapps\common\NeosVR\HeadlessClient\CloudX.Shared.dll

using Newtonsoft.Json;
using System.Text.Json.Serialization;

namespace CloudX.Shared
{
  public class Member
  {
    [JsonProperty(PropertyName = "id")]
    [JsonPropertyName("id")]
    public string UserId { get; set; }

    [JsonProperty(PropertyName = "ownerId")]
    [JsonPropertyName("ownerId")]
    public string GroupId { get; set; }

    [JsonProperty(PropertyName = "quotaBytes")]
    [JsonPropertyName("quotaBytes")]
    public long QuotaBytes { get; set; }

    [JsonProperty(PropertyName = "usedBytes")]
    [JsonPropertyName("usedBytes")]
    public long UsedBytes { get; set; }
  }
}

File: API/Member.js
class Member {
  constructor($b) {
    if (!$b) $b = {};
    this.UserId = $b.id ?? new String();
    this.GroupId = $b.ownerId ?? new String();
    this.QuotaBytes = $b.quotaBytes ?? new Number();
    this.UsedBytes = $b.usedBytes ?? new Number();
  }
}
module.exports = {
  Member
}